### ISSUE TITLE
Overlays support on sidebar

### ISSUE THREAD

**Issue by GuimLe** [files-community/Files#6374]
### What's the Problem?

The overlays are fine only on main panel, not in sidebar.

### Solution/Idea

It is supported on W10 default file explorer, it should be by Files too.

### Alternatives

_Not mentionned_

### Priorities

| Capability | Priority |
| :---------- | :------- |
| To be able to see overlays on sidebar | Must |
| Create a preference to enable these overlays | Could |

### Files Version

Version : 2.0.13.0

### Windows Version

Windows 10 20h2 10.0.19042.1237

**Comment by GuimLe** [comment]
Yes, overlays you get with cloud providers on directories or with SVN, etc...
Like this : 
![image](https://user-images.githubusercontent.com/60383559/136411773-5cc362ef-44d8-4234-8124-2c4597b1e0a8.png)

**Comment by QuaintMako** [comment]
Should be addressed after #12071 .
